code/game/area/space_station_13_areas.dm

```diff
@@ -476,6 +476,10 @@ NOTE: there are two lists of areas in the end of this file: centcom and station
 	airlock_wires = /datum/wires/airlock/service
 	sound_environment = SOUND_AREA_WOODFLOOR
 
+/area/crew_quarters/bar/Initialize(mapload)
+	. = ..()
+	GLOB.bar_areas += src
```

Comment:
is this needed? we keep a global list of areas already right? can't be that expensive to iterate over. I suppose this is good practice